core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/common/nearby/BetaDistributionNearbyRandom.java

```diff
@@ -36,4 +37,19 @@ public int getOverallSizeMaximum() {
         return Integer.MAX_VALUE;
     }
 
+    @Override
+    public boolean equals(Object other) {
+        if (this == other)
+            return true;
+        if (other == null || getClass() != other.getClass())
+            return false;
+        BetaDistributionNearbyRandom that = (BetaDistributionNearbyRandom) other;
+        return Objects.equals(betaDistribution.getAlpha(), that.betaDistribution.getAlpha())
+                && Objects.equals(betaDistribution.getBeta(), that.betaDistribution.getBeta());
```

Comment:
Why not rely on betaDistribution.equal() that implements this?

Reply:
Unfortunately, `BetaDistribution` doesn't seem to implement equals and it is not our class, so I can not add it.